homeassistant/components/hassio/strings.json

```diff
@@ -52,14 +52,14 @@
       "fix_flow": {
         "step": {
           "fix_menu": {
-            "description": "`{reference}` is a filesystem with the name hassos-data and is not the active data disk. This can cause Home Assistant to choose the wrong data disk at system reboot.\n\nUse the 'Rename' option to rename the filesystem to prevent this. Use the 'Adopt' option to make that your data disk and rename the existing one. Alternatively you can move the data disk to the drive (overwriting its contents) or remove the drive from the system.",
+            "description": "At `{reference}`, we detected another active data disk (containing a filesystem `hassos-data` from another Home Assistant installation).\n\nYou need to decide what to do with it, otherwise Home Assistant might choose the wrong data disk at system reboot..\n\nIf you don't want to use this data disk, unplug it from your system. If you leave it plugged in, choose one of the following options:",
```

Comment:
```suggestion
            "description": "At `{reference}`, we detected another active data disk (containing a file system `hassos-data` from another Home Assistant installation).\n\nYou need to decide what to do with it. Otherwise Home Assistant might choose the wrong data disk at system reboot..\n\nIf you don't want to use this data disk, unplug it from your system. If you leave it plugged in, choose one of the following options:",
```